requirements.txt

```diff
@@ -6,7 +6,7 @@ certifi==2017.7.27.1
 cffi==1.10.0
 chardet==3.0.4
 click==6.7
-cryptography==1.7.2
+cryptography
```

Comment:
We lock dependencies. Please use pip-tools to lock dependencies (stated in requirements.in) and generate requirements.txt using pip-compile.

Reply:
I had it there initially and had to remove it because it gave some problems during docker build.

Reply:
Could you please share the error you are getting? IMHO it would be better to fix build error in that case.

Reply:
Sure, here is the Gist of that error - https://gist.github.com/arunkumars08/5d16a7efb2826c87ee796c0e42feda95

Please let me know if I am missing something.

Reply:
It looks like you didn't install dependencies, check installation instructions: https://cryptography.io/en/latest/installation/#rhel-centos

Reply:
As this was fixed, could you lock dependencies using pip-compile?